tests/unit/services/coms-test.js

```diff
@@ -62,3 +62,47 @@ test('#transferMessage calls transferMessage on the appropriate transport servic
 
   service.transferMessage(space, channel, 'hello world');
 });
+
+test('#updateChannelUserList updates the users and connects the channel', function(assert) {
+  const observeMessage = {
+    "@type": "observe",
+    "actor": {
+        "@id": "irc://irc.freenode.net/#kosmos",
+        "@type": "room",
+        "displayName": "#kosmos"
+    },
+    "context": "irc",
+    "object": {
+        "@type": "attendance",
+        "members": [
+            "derbumi",
+            "galfert",
+            "gregkare",
+            "raucao",
+            "slvrbckt"
+        ]
+    },
+    "published": "2017-06-23T15:44:54.383Z"
+  };
+
+  const space = Space.create({
+    protocol: 'IRC',
+    name: 'Freenode',
+    server: { hostname: 'irc.freenode.net' }
+  });
+  const channel = Channel.create({
```

Comment:
Oh wow, totally had the same misconception. Thanks for the link, that's a good post!